packages/app-headless-cms-common/src/types/index.ts

```diff
@@ -344,10 +344,26 @@ export type CmsEditorContentEntry = CmsContentEntry;
 export interface CmsContentEntry {
     id: string;
     modelId: string;
-    savedOn: string;
+    createdOn: string;
     createdBy: CmsIdentity;
-    firstPublishedOn: string | null;
-    lastPublishedOn: string | null;
+    savedOn: string;
+    savedBy: CmsIdentity;
+    modifiedOn: string;
+    modifiedBy: CmsIdentity;
```

Comment:
This can be `CmsIdentity` or `null`.

The `modified...` field is the only nullable field.

Reply:
Thank you @adrians5j.

I fixed the types checking which fields could be `null`.

Nice spot 👍